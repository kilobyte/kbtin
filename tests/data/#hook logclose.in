#hook logclose {#show Log's contents:;#sys cat $0}
#loginputf {> }
#run a //selfpipe
#del 1 #end
#log foo
meow
