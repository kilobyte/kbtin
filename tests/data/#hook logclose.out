meow
Log's contents:
> meow
meow

#SESSION 'a' DIED.
